tests/test_routes.py

```diff
@@ -76,5 +77,16 @@ def test_route_gets_controllers():
     assert Get().route('test/url', 'TestController@show')
     assert Get().route('test/url', '/app.http.test_controllers.TestController@show')
 
+
 def test_route_doesnt_break_on_incorrect_controller():
     assert Get().route('test/url', 'BreakController@show')
+
+
+def test_group_route():
+    routes = group('/example', [
+        Get().route('test/1', 'TestController@show'),
```

Comment:
same thing here. Change this line to `Get().route('/test/1', 'TestController@show')` and get this same test to pass. Good test.

Reply:
Ok